app/components/post_header/post_header.js

```diff
@@ -95,6 +98,19 @@ export default class PostHeader extends PureComponent {
                     </View>
                 </TouchableOpacity>
             );
+        } else if (isGuest) {
+            return (
+                <TouchableOpacity onPress={this.handleUsernamePress}>
+                    <View style={style.indicatorContainer}>
+                        <Text style={style.displayName}>
+                            {this.props.displayName}
+                        </Text>
+                        <GuestTag
+                            theme={this.props.theme}
```

Comment:
if defaulting the `show` prop to false then you need to pass the prop as true here